frontend/src/scenes/insights/InsightTabs/FunnelTab/CorrelationMatrix.tsx

```diff
@@ -0,0 +1,144 @@
+                                    <td>
+                                        {filters.funnel_correlation_details?.type === 'property'
+                                            ? 'Has property'
+                                            : 'Performed event'}
+                                    </td>
+                                    <td>Success</td>
+                                    <td>Dropped off</td>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td className="horizontal-header">Yes</td>
+                                    <td>
+                                        <div className="percentage">
+                                            {percentage(
+                                                correlationDetails.success_count /
+                                                    (correlationDetails.success_count +
+                                                        correlationDetails.failure_count)
+                                            ) || '0.00%'}
+                                        </div>
+                                        {/* TODO: Fix links to person modal */}
+                                        {/* TODO: Handle zero users */}
+                                        <Link to={correlationDetails.success_people_url}>
+                                            {correlationDetails.success_count.toLocaleString()}{' '}
+                                            {pluralize(correlationDetails.success_count, 'user', undefined, false)}
+                                        </Link>
+                                    </td>
+                                    <td>
+                                        <div className="percentage">
+                                            {percentage(
+                                                correlationDetails.failure_count /
+                                                    (correlationDetails.success_count +
+                                                        correlationDetails.failure_count)
+                                            ) || '0.00%'}
+                                        </div>
+                                        <Link to={correlationDetails.failure_people_url}>
+                                            {correlationDetails.failure_count.toLocaleString()}{' '}
+                                            {pluralize(correlationDetails.failure_count, 'user', undefined, false)}
+                                        </Link>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td className="horizontal-header">No</td>
+                                    <td>
+                                        <div className="percentage">?%</div>???
```

Comment:
@neilkakkar can you help me connect these values and the correlation score?

Reply:
@hazzadous maybe you have some context here while @neilkakkar is away?

Reply:
Ah! So `success_count` and `failure_count` come from the API, and `success_total` and `failure_total` come from the funnel: `failure_total` = `stepResults[0].count -  success_total` and `success_total = stepResults[last].count`

Now, 

```
odds_ratio = (
        (success_count + prior_counts)
        * (failure_total - failure_count + prior_counts)
    ) / (
        (success_total - success_count + prior_counts)
        * (failure_count + prior_counts)
    )
```

where prior_counts = 1 (weak prior, and prevents division by zero errors)

Reply:
Also to note: it's slightly more annoying on a funnel with breakdowns, as then the result coming from `stepResults` changes to be a double array, which we ought to sum counts over.

Reply:
And then, to answer your specific question about what should go here: 

Success & No = success_total - success_count
Dropped off & No = failure_total - failure_count

and all percentages should use success_total (or failure_total) , and not `success_count+failure_count` - since that is a percentage over all people who did this specific event, not total people in funnel.

Reply:
Thanks for all this @neilkakkar, super helpful, I've filled the matrix with real values now